### DISCUSSION TITLE
Create an easy way to have threshold alerting

### DISCUSSION THREAD

**Discussion by janpieterz** [datalust/seq-tickets#1085] [Ideas]
We often encounter "single off errors" that we'd like to be alerted about only if they exceed a certain threshold.
The dashboard with a chart of a count for this and an alert works, but is a bit cumbersome to setup for the multitude of things going wrong.  A general error graph helps somewhat but misses the potential granularity.
An option could be some dropdown under an event and the ability to add a threshold alert that will fire a notification app. Basically a "hidden chart".

Some samples for today:
`A transport-level error has occurred when receiving results from the server. (provider: Session Provider, error: 19 - Physical connection is not usable)`
`Reading the request body timed out due to data arriving too slowly. See MinRequestBodyDataRate.`

Neither of these are highly critical, but at the same time if either happens say > 10 times a minute or an hour it'd be an issue we want to be alerted about.

**Comment by nblumhardt** [comment]
Thanks for the suggestion, we'll give this one some thought :+1:

**Comment by larenelg** [comment]
Hi @janpieterz 

Thank you for your feedback!

We're currently working on improving alerting UX in Seq, and just wanted to let you know we're planning on removing the need for a chart before you can create an alert, and are experimenting with creating alerts from various places in Seq e.g. from individual events, searches, and queries.

We hope to have more details posted in 2021.
